doc-content/enterprise-only/using-ansible-rhdm-in-ONAP/technology-and-definitions-con.adoc

```diff
@@ -0,0 +1,25 @@
+[id='technology-and-definitions-con']
+= Technology and definitions
+
+== What is ONAP?
+
```

Comment:
For the sake of being consistent with our other user story types, can we not have questions as titles? (applies to all such titles in this doc)

Reply:
Changed as suggested. 

There were two occurrences of questions as titles, changed them both to:
- What is ONAP? to Open Network Automation Platform,
- What is policy? to Policy.